src/Dashboard/BlogView.jsx

```diff
@@ -0,0 +1,20 @@
+import "../App.css";
+import { useParams } from "react-router-dom";
+import { TextBox } from "../components/TextBox";
+import { useFetch } from "../constants/useFetch";
+
+const BlogView = () => {
+    const { id } = useParams();
+    const [data, isLoading, error] =useFetch(`https://jsonplaceholder.typicode.com/posts/${id}`, 'get');
```

Comment:
https://jsonplaceholder.typicode.com

Host can be saved and come from constant and eventually from .env files